Nodejs/Product/Nodejs/Debugger/NodeDebugger.cs

```diff
@@ -1239,6 +1240,31 @@ public NodeDebugger(Uri debuggerEndpointUri, int id)
             return true;
         }
 
+        /// <summary>
+        /// Get the original file name to map to.
+        /// </summary>
+        /// <param name="javaScriptFileName">JavaScript compiled file.</param>
+        /// <param name="stackFrame">The current stack frame.</param>
+        internal string GetOriginalFileName(string javaScriptFileName, NodeStackFrame stackFrame) {

```

Comment:
Not a fan of conflating the debugging logic with the source mapping logic... probably best to move the core logic to SourceMapper.cs (which would accepts line/column arguments), and use a helper here if necessary.